lms/envs/common.py

```diff
@@ -2146,8 +2149,8 @@
     # API access administration
     'openedx.core.djangoapps.api_admin',
 
-    # Verified Track Content Cohorting
-    'verified_track_content',
+    # Verified Track Content Cohorting (Beta feature that will hopefully be removed)
```

Comment:
Is there a TNL ticket for this cleanup task?  Consider adding the ticket number to this comment.